### ISSUE TITLE
more font options

### ISSUE THREAD

**Issue by mspykerez** [brittyazel/EnhancedRaidFrames#69]
There is font options for Indicators, but what about the default player names, etc, ? Would be nice to choose the font and size for those too.

**Comment by brittyazel** [comment]
Yeah so far we've only allowed for changing the font and size on the text that we're adding to the default frames, not the text that already exists. There's a number of difficulties with modifying existing functionality and elements of the frames, and if I do a bad job it will result in lots and lots of taint for everyone. It's probably possibly, but we just need to tread lightly when we start injecting changes into Blizzard's code.
